incubator/etcd/templates/statefulset.yaml

```diff
@@ -47,6 +47,9 @@ spec:
           value: {{ .Values.replicas | quote }}
         - name: SET_NAME
           value: {{ template "etcd.fullname" . }}
+{{- if .Values.env }}
+{{ toYaml .Values.env | indent 8 }}
+{{- end }}
```

Comment:
I will suggest doing this through a range that looks like this:
```yaml
{{- range $key, $value := .Values.env }}
            - name: "{{ $key }}"
              value: "{{ $value }}"
{{- end }}
```
So that `values.yaml` looks less verbose and cleaner when adding env variables.

Reply:
I have changed it to this. I was doubting myself but saw both ways being used in the charts here so didn't know which way was prefered